### PR TITLE
Fix manifest collection in failed test

### PR DESCRIPTION
**Pull request by smoshiur1237**
This will add the capability to add manifest in failed test and will improve the debugging capability.

### CODE REVIEW

File: jenkins/scripts/files/run_fetch_logs.sh [hunk 1]
```diff
@@ -58,6 +54,57 @@ fetch_k8s_logs()
...
+      m3dataclaim
+      m3datatemplate
+    )
+
+    NAMESPACES="$(kubectl --kubeconfig="${kconfig}" get namespace -o jsonpath='{.items[*].metadata.name}')"
+    for NAMESPACE in ${NAMESPACES}; do
+      for kind in "${manifests[@]}"; do
+        mkdir -p "${LOGS_DIR}/${dir_name}/${kind}"
+        for name in $(kubectl --kubeconfig="${kconfig}" get -n "${NAMESPACE}" -o name "${kind}" || true); do
+          kubectl --kubeconfig="${kconfig}" get -n "${NAMESPACE}" -o yaml "${name}" | tee "${LOGS_DIR}/${dir_name}/${kind}/$(basename "${name}").yaml" || true
```
Comment (mboukhalfa) [inline jenkins/scripts/files/run_fetch_logs.sh:92]: if we have the same kind in different namespace then would those get overridden ?

Comment (tuminoid) [reply jenkins/scripts/files/run_fetch_logs.sh:92]: Good catch, and we probably have the same issue in `fetch_manifests.sh` where this was copied from. Namespace must be part of the filename or path, or you need to fetch all at once, not per namespace.

/hold

Comment (smoshiur1237) [reply jenkins/scripts/files/run_fetch_logs.sh:92]: If I use the Namespace as part of the path, that should solve our issue, isn't it?

Comment (tuminoid) [reply jenkins/scripts/files/run_fetch_logs.sh]: You need to put the namespace to the `mkdir` as well.

**Comment by tuminoid** [comment]
/test metal3-ubuntu-e2e-integration-test-main metal3-centos-e2e-integration-test-main 

One of them should pass right now, and another fail, so it'd be a good test.

**Review by tuminoid — COMMENTED** [review]
/approve

Should be fine, let's check the tests logs when completed.

**Comment by tuminoid** [comment]
> /approve
> 
> Should be fine, let's check the tests logs when completed.

No manifests in the centos failure case? https://jenkins.nordix.org/view/Metal3/job/metal3-centos-e2e-integration-test-main/329/

**Comment by smoshiur1237** [comment]
> > /approve
> > Should be fine, let's check the tests logs when completed.
> 
> No manifests in the centos failure case? https://jenkins.nordix.org/view/Metal3/job/metal3-centos-e2e-integration-test-main/329/

Yes I know the reason. The file I have changed was for dev-env ansible test run. For e2e We are using the dynamic worker workflow where it has the same fetch log for e2e test. So I will update this PR to add the same changes in dynamic worker workflow

**Review by Rozzii — COMMENTED** [review]
/lgtm

**Pull request merged**